identity/src/main/java/com/android/identity/util/EngagementTypeDef.kt

```diff
@@ -0,0 +1,48 @@
+package com.android.identity.util
+
+import com.android.identity.util.EngagementTypeDef.TypeDef.ENGAGEMENT_METHOD_NFC_NEGOTIATED_HANDOVER
+import com.android.identity.util.EngagementTypeDef.TypeDef.ENGAGEMENT_METHOD_NFC_STATIC_HANDOVER
+import com.android.identity.util.EngagementTypeDef.TypeDef.ENGAGEMENT_METHOD_NOT_ENGAGED
+import com.android.identity.util.EngagementTypeDef.TypeDef.ENGAGEMENT_METHOD_QR_CODE
+import com.android.identity.util.EngagementTypeDef.TypeDef.ENGAGEMENT_METHOD_REVERSE
+import com.android.identity.util.EngagementTypeDef.TypeDef.ENGAGEMENT_METHOD_UNATTENDED
+
```

Comment:
This enum class brings all the `ENGAGEMENT_METHOD_` ints that used to live in `VerificationHelper` into its own cohesive and decoupled class. These types surely are used in `VerificationHelper` but they are not strictly bound to only the functionality found inside that class.